### PR TITLE
chore: rename authority -> validator

### PR DESCRIPTION
**Pull request by huitseeker**
To get us started!

### CODE REVIEW

File: doc/src/build/authorities.md [hunk 1]
```diff
@@ -1,33 +1,33 @@
 ---
```
Comment (oxade) [inline doc/src/build/authorities.md:1]: Probably want to rename this file to `validators.md`

Comment (huitseeker) [reply doc/src/build/authorities.md:1]: Voluntarily, I've left that to a further PR along with the code changes. This makes review easier:
https://github.com/MystenLabs/sui/pull/1398/files#diff-4bf78b90fcc082845477a30625a3a7d864cac80492da8e1621ba853ff5e19a30R11

Comment (oxade) [reply doc/src/build/authorities.md:1]: Gotcha!

**Review by Clay-Mysten — APPROVED** [review]
Thank you so much, Francois! I've fixed instances of "an validator" in direct commits to your PR. Thanks again!

**Pull request merged**